caluma/caluma_form/schema.py

```diff
@@ -169,6 +169,7 @@ def resolve_type(cls, instance, info):
 
 class Option(FormDjangoObjectType):
     meta = generic.GenericScalar()
+    is_hidden = QuestionJexl(required=True)
```

Comment:
Ackchyually not a `QuestionJexl` but I will let it slide.

Reply:
Actually it is. Option uses the same JEXL class the questions use.